wavefront_opentracing_sdk/span.py

```diff
@@ -134,7 +152,22 @@ def _do_finish(self, duration_time):
                 return
             self.duration_time = duration_time
             self._finished = True
-        self.tracer.report_span(self)
+        # perform another sampling for duration based samplers
+        if not self._force_sampling and not self._context.is_sampled() \
```

Comment:
Can you use parenthesis for clarity:
```
not self._force_sampling and (not self._context.is_sampled() or not self._context.get_sampling_decision())
```

Reply:
It's equivalent to `not (self._context.get_sampling_decision() and (self._force_sampling or self._context.is_sampled())`

Reply:
What @vikramraman proposed would be equivalent to `not (self.force_sampling() or (self.context.is_sampled and self._context.get_sampling_gecision()))`